SoftLayer/CLI/hardware/toggle_ipmi.py

```diff
@@ -0,0 +1,23 @@
+"""Toggle the IPMI interface on and off."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import helpers
+
+
+@click.command()
+@click.argument('identifier')
+@click.option('--enabled',
```

Comment:
You can use a [Boolean Flag](https://click.palletsprojects.com/en/7.x/options/#boolean-flags) here, so the user doesn't have to actually type in `enabled=True`.

See https://github.com/softlayer/softlayer-python/blob/master/SoftLayer/CLI/user/edit_permissions.py#L13 for an example.

Reply:
I fixed it as following :
```
@click.option('--enable/--disable', default=True,
              help="Whether enable (DEFAULT) or disable the interface.")
```